### ISSUE TITLE
cudnn accelerate is not work

### ISSUE THREAD

**Issue by vegetable09** [ChineseGLUE/ChineseGLUE#85]
Using tensorflow/tensorflow:1.12-gpu
Although this docker image doesn't installed cudnn, the msraner_task baseline still work. the only problem is it's too slow.

run on my 1080Ti 11GB GPU. But the speed doesn't change, like following:
INFO:tensorflow:global_steps = 10230, loss = 0.053016827 (109.679 sec)
INFO:tensorflow:global_step/sec: 1.98741
INFO:tensorflow:examples/sec: 31.7985

So I guess I can use CUDNN to accelerate the training. But after copy cudnn7.0.5 libraries and includes to /usr/local/cuda/ corresponding directionary and chmod them, the speed of training doesn't up.
